src/Route/Estimator.test.ts

```diff
@@ -28,102 +28,141 @@ describe('Estimator', () => {
 
 	describe('#estimateCarDirection', () => {
 		it('should correctly build car direction when distance is from 0 to 2 km', () => {
-			const direction: Direction = Estimator.estimateCarDirection(1000, origin, destination);
-			chai.expect(direction.mode).to.equal(TransportMode.CAR);
-			chai.expect(direction.polyline).to.equal('_c`|@_c`|@_ibE_ibE');
-			chai.expect(direction.distance).to.equal(1800);
-			chai.expect(direction.duration).to.equal(133);
-			chai.expect(direction.avoid).to.deep.equal([]);
-			chai.expect(direction.isoCodes).to.deep.equal([]);
-			chai.expect(direction.source).to.equal('estimator');
-			chai.expect(direction.routeId).to.be.null('Expect null');
+			const direction: Direction | null = Estimator.estimateCarDirection(1000, origin, destination);
+			chai.expect(direction).to.not.be.null('Dont expect null');
+			if (direction) {
+				chai.expect(direction.mode).to.equal(TransportMode.CAR);
```

Comment:
I would use the `!` operator that ensures that variable is not null (it could be null in runtime though) instead of `if` condition.
`chai.expect(direction!.mode).to.equal(TransportMode.CAR);`

Reply:
force pushed